### PR TITLE
Moved PhoneNumbers project to Nox.Reference.World.

### PR DESCRIPTION
**Pull request by AndreyDegtyarev**
Introduced PhoneCarrier entity.
Moved PhoneNumberService to Nox.Reference.World

### CODE REVIEW

File: src/Nox.Reference.Data.World/Generated/Currencies.cs [hunk 1]
```diff
@@ -1,155 +1,153 @@
...
+	VietnameseDong,
+	NiVanuatuVatu,
+	SamoanTālā,
+	EastCaribbeandollar,
+	CFAFranc,
+	CFPFranc,
+	YemeniRial,
+	SouthAfricanRand,
+	ZambianKwacha,
}
```
Comment (ILoveMyTortoise) [inline src/Nox.Reference.Data.World/Generated/Currencies.cs:153]: Why this changes each pr? Why it has comma in the end? I think we have a problem

Comment (AndreyDegtyarev) [reply src/Nox.Reference.Data.World/Generated/Currencies.cs:153]: It should be excluded from solution so far. I added enum generator which is invoked during seeder.
